Replace STATIC_ITEM with PSTRING_ITEM for print count and completed prints

Haxk20 opened the pull request:
# Description

While looking at the info menu on my printer the numbers were not even separated from the text.
This as you can imagine does not look very good to an user.

This was an side effect of commit bff0c68 which deleted the previously used definition and replaced
it with STATIC_ITEM which is wrong.

### Benefits

Makes the info menu little prettier.

### Fixes

Fixes commit bff0c68

sjasonsmith commented on Marlin/src/lcd/language/language_en.h:
```diff
@@ -506,8 +506,8 @@ namespace Language_en {
-    PROGMEM Language_Str MSG_INFO_COMPLETED_PRINTS         = _UxGT("Completed");
+    PROGMEM Language_Str MSG_INFO_PRINT_COUNT              = _UxGT("Print Count: ");
+    PROGMEM Language_Str MSG_INFO_COMPLETED_PRINTS         = _UxGT("Completed: ");
     PROGMEM Language_Str MSG_INFO_PRINT_TIME               = _UxGT("Total Print Time");
     PROGMEM Language_Str MSG_INFO_PRINT_LONGEST            = _UxGT("Longest Job Time");
     PROGMEM Language_Str MSG_INFO_PRINT_FILAMENT           = _UxGT("Extruded Total");
```
Do these not appear in the menu? Why do some need colons and some do not?

sjasonsmith replied:
Do some already have colons added in the code which uses the string, rather than here?

Haxk20 replied:
These ones behave a little different.
Here a new line with ">" and the characters we wish to show is drawn. Thus not needing : from visual perspective.

Haxk20 replied:
Tho if needed the lines i changed could be changed to behave like the ones you selected here in code.

sjasonsmith commented:
Could you share a picture of the menu, so everybody can see how this is displayed?

Haxk20 commented:
@sjasonsmith Sure thing

Haxk20 commented:
![Info Menu with this PR](https://user-images.githubusercontent.com/16738302/92962373-2952de80-f471-11ea-84de-88d0f737f085.jpg)
@sjasonsmith

sjasonsmith commented:
Thanks! I see now that those other ones use two lines, so the colon is less important.

It is possible that the colons should be added by the calling code, rather than in the language strings. This might make them more usable for different display types which present the data differently. I'm not familiar enough with the various displays and string code to know which would be preferred, but I'm sure @thinkyhead will.

Haxk20 commented:
That is definetly possible. Tho i havent found any recent PR or commit removing such code from M1001

Haxk20 committed:
menu_info: Replace STATIC_ITEM with PSTRING_ITEM for print count and completed prints

With STATIC_ITEM the two strings would render number right after them
without any space or ":".
This looked horrible.

This fixes bff0c68 (Consistent static/value item macros)

Haxk20 commented:
@sjasonsmith This is the proper way of fixing this. Please if you want review it. Thank you.

thinkyhead committed:
Restore custom info items

Haxk20 commented:
@thinkyhead should I drop my commit orrrr ? Cause you based the partial revert on top of my commit.
Or do you want to merge it as it is now ?

thinkyhead committed:
Bring back VALUE_ITEM macros
